carbonmark/hooks/useFetchProjects.ts

```diff
@@ -12,19 +24,24 @@ import { useEffect, useRef } from "react";
  *
  * @note we may need to do the same for other generated hooks
  */
-export const useFetchProjects = (
-  ...args: Parameters<typeof useGetProjects<GetProjectsQueryResponse>>
-) => {
-  const res = useGetProjects(...args);
-  const prevParams = useRef(args[0]);
-  const currParams = args[0];
+export const useFetchProjects = () => {
+  const router = useRouter();
+
+  /** convert all string arrays to comma separated string as per the api's expectation */
+  const mappedParams = mapValues(
+    router.query,
+    pipe(joinArray, emptyToUndefined)
+  );
+
+  const projects = useSWR<TData, unknown, string | null>(`/projects`, {
```

Comment:
@0xemc there seems to be an issue with the params passed through to the `useGetProjects` hook. I originally by mistake had modified the `useGetProjects` file without realising it was auto generated and @psparacino pointed it out. I've since just moved the logic into this hook and am using `useSWR` directly instead of the useGetProjects hook as I couldn't get it to work as expected to fix the issue. 

Happy to come circle back to looking at this and trying to get the autogenerated hook working as expected when it's less time sensitive as the ticket is a high priority for COP.